core/eth/tx.go

```diff
@@ -588,6 +588,27 @@ func (t *Transactor) GetCurrentQuorumBitmapByOperatorId(ctx context.Context, ope
 }
 
 func (t *Transactor) GetQuorumBitmapForOperatorsAtBlockNumber(ctx context.Context, operatorIds []core.OperatorID, blockNumber uint32) ([]*big.Int, error) {
+	if len(operatorIds) == 0 {
+		return []*big.Int{}, nil
+	}
+	// When there is just one operator, we can get result by a single RPC with
+	// getQuorumBitmapsAtBlockNumber() in OperatorStateRetrievercontract (v.s. 2
+	// RPCs in the general case)
+	if len(operatorIds) == 1 {
+		byteId := [32]byte(operatorIds[0])
+		bitmap, err := t.Bindings.OpStateRetriever.GetQuorumBitmapsAtBlockNumber(&bind.CallOpts{
+			Context: ctx,
+		}, t.Bindings.RegCoordinatorAddr, [][32]byte{byteId}, blockNumber)
+		if err != nil {
+			if err.Error() == "RegistryCoordinator.getQuorumBitmapIndexAtBlockNumber: no bitmap update found for operatorId at block number" {
```

Comment:
does this equality match the err properly? have we checked this is the exact string that gets returned for this error?

Reply:
I think @0x0aa0 is helping check the details, but haven't tested yet

Reply:
Ok, I'm sure it's "execution reverted: RegistryCoordinator.getQuorumBitmapIndexAtBlockNumber: no bitmap update found for operatorId at block number"